assets/js/components/channels/ChannelBar.jsx

```diff
@@ -1,9 +1,35 @@
 import React from 'react';
 import NavPointTriangle from '../common/NavPointTriangle';
 import BarIcon from '../../../img/channels/channel-bar-icon.svg'
+import { getIntegrationTypeForFlows } from '../../util/flows'
 import { Typography } from 'antd';
 const { Text } = Typography;
 import { Link } from 'react-router-dom';
+import AdafruitIcon from '../../../img/channels/adafruit.png';
+import AwsIcon from '../../../img/channels/aws.png';
+import AzureIcon from '../../../img/channels/azure.png';
+import CargoIcon from '../../../img/channels/cargo.png';
+import CayenneIcon from '../../../img/channels/cayenne.png';
+import DatacakeIcon from '../../../img/channels/datacake.png';
+import HttpIcon from '../../../img/channels/http.png';
+import MqttIcon from '../../../img/channels/mqtt.png';
+import TagoIcon from '../../../img/channels/tago.png';
+import UbidotsIcon from '../../../img/channels/ubidots.png';
+import GoogleSheetIcon from '../../../img/google-channel.svg'
+
+const imgMap = {
+  adafruit: AdafruitIcon,
+  aws: AwsIcon,
+  azure: AzureIcon,
+  cargo: CargoIcon,
+  cayenne: CayenneIcon,
+  datacake: DatacakeIcon,
+  http: HttpIcon,
+  mqtt: MqttIcon,
+  tago: TagoIcon,
+  ubidots: UbidotsIcon,
+  googlesheets: GoogleSheetIcon
+}
```

Comment:
since we use he same object for `ChannelNode` too, maybe we can put this in a shared constant file and reuse for both?

Reply:
good point, ty!